app/src/userpages/components/ProfilePage/index.jsx

```diff
@@ -99,43 +97,48 @@ export class ProfilePage extends Component<Props, State> {
                             },
                         }}
                     />
-                    <div className={cx('container', styles.containerOverrides)}>
-                        <TOCPage title={I18n.t('userpages.profilePage.pageTitle')}>
-                            <TOCPage.Section id="profile" title={I18n.t('userpages.profilePage.profile.title')}>
-                                <ProfileSettings />
-                            </TOCPage.Section>
-                            <TOCPage.Section
-                                id="api-keys"
-                                title={I18n.t('userpages.profilePage.apiCredentials.title')}
-                                linkTitle={I18n.t('userpages.profilePage.apiCredentials.linkTitle')}
-                            >
-                                <APICredentials />
-                            </TOCPage.Section>
-                            <TOCPage.Section
-                                id="ethereum-accounts"
-                                title={I18n.t('userpages.profilePage.ethereumAddress.title')}
-                                linkTitle={I18n.t('userpages.profilePage.ethereumAddress.linkTitle')}
-                            >
-                                <IdentityHandler />
-                            </TOCPage.Section>
-                            <TOCPage.Section
-                                id="private-keys"
-                                title={I18n.t('userpages.profilePage.ethereumPrivateKeys.title')}
-                                linkTitle={I18n.t('userpages.profilePage.ethereumPrivateKeys.linkTitle')}
-                            >
-                                <IntegrationKeyHandler />
-                            </TOCPage.Section>
-                            <TOCPage.Section
-                                id="delete-account"
-                                title={I18n.t('userpages.profilePage.deleteAccount.title')}
-                                linkTitle={I18n.t('userpages.profilePage.deleteAccount.linkTitle')}
-                            >
-                                <DeleteAccount />
-                            </TOCPage.Section>
-                        </TOCPage>
-                    </div>
+                )}
+            >
+                <Helmet title={`Streamr Core | ${I18n.t('userpages.title.profile')}`} />
```

Comment:
Probably for a cleanup PR of its own- we're moving to 'page_title | Streamr Core' on the Docs project since when tabs are small, you want the unique information left-most. Aiming to be consistent eventually : )

Reply:
Ok, good to know! I'll leave it now and let's fix all at once.